python/ctsm/test/test_unit_run_tower.py

```diff
@@ -18,14 +18,14 @@
 
 # pylint: disable=wrong-import-position
 from ctsm import unit_testing
-from ctsm.site_and_regional.run_neon import check_neon_listing
+from ctsm.site_and_regional.run_tower import check_neon_listing
 
```

Comment:
do we also need to check plumber listing here?  Beyond my pay grade

Reply:
This checks that the NEON data is properly downloaded from the NEON listing csv file; PLUMBER doesn't use a comparable data download method, so I would say it's not needed.